### PR TITLE
Update default `run-name` triggered by `pull_request` event

### LINKED ISSUE #32266: Default `run-name` of workflow run triggered by `pull_request`
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#run-name

### What part(s) of the article would you like to see updated?

`run-name`

> GitHub displays the workflow run name in the list of workflow runs on your repository's "Actions" tab. If `run-name` is omitted or is only whitespace, then the run name is set to event-specific information for the workflow run. For example, for a workflow triggered by a `push` or `pull_request` event, it is set as the commit message.

But from what I seen in "Actions" tab, the default `run-name` of workflow runs triggered by `pull_request` is the corresponding PR title, not commit message.

For example
- https://github.com/muzimuzhi/hello-github-actions/actions?query=event%3Apull_request
- https://github.com/latex3/latex3/actions?query=event%3Apull_request

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by muzimuzhi**
### Why:

Closes: #32266

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Update docs according to current behavior of GitHub Actions, in which the default `run-name` of workflow runs triggered by `pull_request` event, found on "Actions" tab, uses PR title, not commit message.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by muzimuzhi** [comment]
https://github.com/github/docs/actions/runs/8470303578/job/23207709363?pr=32267 failed with a TLS handshake timeout:

(newlines and indents added by me)
```
Error: buildx failed with: ERROR:
  failed to solve: node:20-alpine@sha256:bf77dc26e48ea95fca9d1aceb5acfa69d2e546b765ec2abfb502975f1a2d4def:
  failed to resolve source metadata for docker.io/library/node:20-alpine@sha256:bf77dc26e48ea95fca9d1aceb5acfa69d2e546b765ec2abfb502975f1a2d4def:
  failed to do request: Head "https://registry-1.docker.io/v2/library/node/manifests/sha256:bf77dc26e48ea95fca9d1aceb5acfa69d2e546b765ec2abfb502975f1a2d4def":
  net/http: TLS handshake timeout
```

**Comment by nguyenalex836** [comment]
@muzimuzhi Thanks so much for opening a PR! I'll get this triaged for review ✨

**Review by R0nni333 — COMMENTED** [review]
.

File: content/actions/using-workflows/workflow-syntax-for-github-actions.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ You must store workflow files in the `.github/workflows` directory of your repos
 
 ## `run-name`
 
-The name for workflow runs generated from the workflow. {% data variables.product.prodname_dotcom %} displays the workflow run name in the list of workflow runs on your repository's "Actions" tab. If `run-name` is omitted or is only whitespace, then the run name is set to event-specific information for the workflow run. For example, for a workflow triggered by a `push` or `pull_request` event, it is set as the commit message.
+The name for workflow runs generated from the workflow. {% data variables.product.prodname_dotcom %} displays the workflow run name in the list of workflow runs on your repository's "Actions" tab. If `run-name` is omitted or is only whitespace, then the run name is set to event-specific information for the workflow run. For example, for a workflow triggered by a `push` or `pull_request` event, it is set as the commit message or title of pull request, respectively.
```
Comment (SiaraMist) [inline content/actions/using-workflows/workflow-syntax-for-github-actions.md:33]:
```suggestion
The name for workflow runs generated from the workflow. {% data variables.product.prodname_dotcom %} displays the workflow run name in the list of workflow runs on your repository's "Actions" tab. If `run-name` is omitted or is only whitespace, then the run name is set to event-specific information for the workflow run. For example, for a workflow triggered by a `push` or `pull_request` event, it is set as the commit message or the title of the pull request.
```

**Review by SiaraMist — APPROVED** [review]
Thanks so much for updating the docs @muzimuzhi, and for providing examples to verify your changes in the issue you opened. ✨

I just made a small edit to match our style guide, and I'll go ahead and get this merged.

**Pull request merged**